src/presentation.js

```diff
@@ -466,10 +492,70 @@ export default class Presentation extends React.Component {
 
         <Slide transition={['fade']} bgColor="primary" textColor="secondary">
           <Image src={updatingServiceWorker} />
-          <Link href="https://deanhume.com/displaying-a-new-version-available-progressive-web-app/">deanhume.com/displaying-a-new-version-available-progressive-web-app/</Link>
+          <Link href="https://deanhume.com/displaying-a-new-version-available-progressive-web-app/">
+            deanhume.com/displaying-a-new-version-available-progressive-web-app/
+          </Link>
         </Slide>
-        <Slide transition={['fade']} bgColor="primary" textColor="secondary">
+        <Slide>
+          <Heading>Reaching the user</Heading>
+          <List>
+            {this.renderListItem('keep it web only')}
+            {this.renderListItem('Deploy it to an App Store!')}
+          </List>
+        </Slide>
+
+        <Slide>
+          <Heading>Store Deployment</Heading>
+          <List>
+            {this.renderListItem('Play Store integration since 2018')}
+            {this.renderListItem('no iOS support tho')}
```

Comment:
Capital first letter for consistency please, also maybe use "though" instead :smile:

Reply:
NEVER